src/sss_client/subid/sss_subid.c

```diff
@@ -0,0 +1,206 @@
+/*
+    Copyright (C) 2021 Red Hat
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <stdlib.h>
+#include <string.h>
+#include <shadow/subid.h>
+#include "sss_cli.h"
+
+
+/* Find all subid ranges delegated to a user.
```

Comment:
Can you use doxygen format to comment the APIs? Example: https://github.com/SSSD/sssd/blob/89a40e77a1477a3957f4ddc47890eaecbc4d5c7c/src/confdb/confdb.c#L2530

I think it only applies to the three functions in this file but I might be mistaken.

Reply:
This is an implementation of https://github.com/shadow-maint/shadow/blob/d4b6d1549b2af48ce3cb6ff78d9892095fb8fdd9/lib/prototypes.h#L271
Does it make sense to copy&paste doxygen comments from original API definition?

Reply:
No, but it might be nice to provide link to this doc.

Reply:
Fixed.